src/TestSuite/StringCompareTrait.php

```diff
@@ -43,6 +54,15 @@ public function assertSameAsFile($path, $result)
     {
         $path = $this->_compareBasePath . $path;
```

Comment:
@markstory Currently, you can't provide an absolute path. Well you can, but that would fail. @ravage84 asked about it on the docs PR : https://github.com/cakephp/docs/pull/3720#discussion_r51712427

One possible solution would be to add a third param ``$absolutePath`` as bool, defaulted to ``false`` to preserve backward compatibility. If set to ``true``, ``$_compareBasePath`` should not be prepended.